build.gradle.kts

```diff
@@ -11,6 +11,8 @@ plugins {
     id("io.spring.dependency-management") version "1.0.9.RELEASE" apply false
     kotlin("jvm") version "1.3.71" apply false
     kotlin("plugin.spring") version "1.3.71" apply false
+    id("org.jetbrains.kotlin.plugin.jpa") version "1.3.71"
+    groovy
```

Comment:
Same, only needed in coffee-backend? Or needed for general Travis build?

Reply:
Needed for groovy to work at all :/ Tried adding it only in the lower build file, but then nothing would build

Reply:
Edit: Apparently that was wrong. Removed it in the parent build file, and still works 🤔